app/components/build-header.js

```diff
@@ -42,5 +42,9 @@ export default Ember.Component.extend({
 
   elapsedTime: Ember.computed('item.startedAt', 'item.finishedAt', 'item.duration', function () {
     return durationFrom(this.get('item.startedAt'), this.get('item.finishedAt'));
+  }),
+
+  urlGitHubBranch: Ember.computed('item.repo.slug', 'item.branch.name', function () {
+    return githubBranch(this.get('item.repo.slug'), this.get('item.branch.name'));
   })
```

Comment:
This is perfect, but I thought you might be interested to see a peek of the future, available as the [`ember-computed-decorators`](https://github.com/rwjblue/ember-computed-decorators) addon, which would change this with the proposed ES2017 (maybe?) decorators feature. If you haven’t already heard:

```
@computed('item.repo.slug', 'item.branch.name')
urlGitHubBranch(slug, branchName) {
  return gitHubBranch(slug, branchName);
}
```

I think it’ll be nice to not have that duplication of specifying the dependent keys in the computed property header and then having to fetch them within the function.